Update docs (#1283)
python-hyper/h2: docs/source/basic-usage.rst

happens as we feed it data.

To make HTTP/2 connections, we need a tool that knows how to speak HTTP/2.
You can simply use `curl`_ or install a Python tool used throughout this
tutorial. In your Python environment, run ``pip install hyper`` (Make sure to
use ``Python < 3.10`` or install it in separate environment). This will
install a Python command-line HTTP/2 tool called ``hyper``. To confirm that
it works, try running this command and verifying that the output looks similar

> to the one shown below:

Reviewer: Since the `hyper` project itself is archived / no longer supported / not maintained, I would suggest to remove the mention of it in this paragraph entirely. `curl` is sufficiently modern nowadays and can be the only mention here.

Reviewer: Do you agree with removing `hyper` from this paragraph?

     'origin': '10.0.0.2',
     'url': 'https://nghttp2.org/httpbin/get'}

Equivalent code with curl would look like this:

.. code-block:: console

    $ curl --http2 https://nghttp2.org/httpbin/get

> To use it with our server though, you will need to invoke it with a different

Reviewer: ```suggestion
To use it with a `python-hyper/h2`-based server, you can invoke it with a different
```

        handle(sock.accept()[0])

Running that in one shell, in your other shell you can run
``hyper --h2 GET http://localhost:8080/``. For the ``curl`` use

> ``curl -v --http2-prior-knowledge http://localhost:8080/`` command.

Reviewer: Same as above, please remove / replace the usage of `hyper` with `curl` here.

Author: Did update it, for some reason forgot to push changes, should be good now
